README.md

```diff
@@ -24,9 +24,13 @@ or rename a template from the templates dir
 cp node_modules/eslint-config-digitalbazaar/templates/node.js ./.eslintrc.js
 ```
 
-Instructions:
-- [ ] inside your .eslintrc.js add "eslint-config-digitalbazaar" to your extends array
-- [ ] then add envs node, browser, or mocha to the relevant dirs
+## Usage
+We export multiple config files that can be used to extend your rules.
+
+All rules can accessed via their shorthand: `digitalbazaar`
```

Comment:
if we're promoting use of shorthand, docs should reflect that throughout.

Reply:
this one is subjective so recheck the docs and mark as resolved if you like it.